sparse/src/KokkosSparse_sptrsv.hpp

```diff
@@ -202,52 +235,56 @@ void sptrsv_symbolic(ExecutionSpace &space, KernelHandle *handle,
       KokkosSparse::Experimental::SPTRSVAlgorithm::SPTRSV_CUSPARSE) {
 #ifdef KOKKOSKERNELS_ENABLE_TPL_CUSPARSE
     if constexpr (std::is_same_v<ExecutionSpace, Kokkos::Cuda>) {
-      using RowMap_Internal = Kokkos::View<
-          typename lno_row_view_t_::const_value_type *,
-          typename KokkosKernels::Impl::GetUnifiedLayout<
-              lno_row_view_t_>::array_layout,
-          typename lno_row_view_t_::device_type,
-          Kokkos::MemoryTraits<Kokkos::Unmanaged | Kokkos::RandomAccess> >;
-
-      using Entries_Internal = Kokkos::View<
-          typename lno_nnz_view_t_::const_value_type *,
-          typename KokkosKernels::Impl::GetUnifiedLayout<
-              lno_nnz_view_t_>::array_layout,
-          typename lno_nnz_view_t_::device_type,
-          Kokkos::MemoryTraits<Kokkos::Unmanaged | Kokkos::RandomAccess> >;
-
-      using Values_Internal = Kokkos::View<
-          typename scalar_nnz_view_t_::const_value_type *,
-          typename KokkosKernels::Impl::GetUnifiedLayout<
-              scalar_nnz_view_t_>::array_layout,
-          typename scalar_nnz_view_t_::device_type,
-          Kokkos::MemoryTraits<Kokkos::Unmanaged | Kokkos::RandomAccess> >;
-
       RowMap_Internal rowmap_i   = rowmap;
       Entries_Internal entries_i = entries;
       Values_Internal values_i   = values;
 
-      typedef typename KernelHandle::SPTRSVHandleType sptrsvHandleType;
-      sptrsvHandleType *sh = handle->get_sptrsv_handle();
-      auto nrows           = sh->get_nrows();
-
-      KokkosSparse::Impl::sptrsvcuSPARSE_symbolic<
-          ExecutionSpace, sptrsvHandleType, RowMap_Internal, Entries_Internal,
-          Values_Internal>(space, sh, nrows, rowmap_i, entries_i, values_i,
-                           false);
+      // typedef typename KernelHandle::SPTRSVHandleType sptrsvHandleType;
+      // sptrsvHandleType *sh = handle->get_sptrsv_handle();
+      // auto nrows           = sh->get_nrows();
+
+      std::string label = "KokkosSparse::sptrsv[TPL_CUSPARSE,"
+	+ Kokkos::ArithTraits<typename scalar_nnz_view_t_::non_const_value_type>::name() + "]";
+      Kokkos::Profiling::pushRegion(label);
+      // KokkosSparse::Impl::sptrsvcuSPARSE_symbolic<
+      //     ExecutionSpace, sptrsvHandleType, RowMap_Internal, Entries_Internal,
+      //     Values_Internal>(space, sh, nrows, rowmap_i, entries_i, values_i,
+      //                      false);
```

Comment:
Should these commented lines be deleted?

Reply:
Done